### PR TITLE
added levenshtein.1.1.3

### PR DESCRIPTION
**Pull request by camlspotter**

### CODE REVIEW

File: packages/levenshtein/levenshtein.1.1.3/descr [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Levenshtein distance algorithm for general array.
+Levenshtein distance algorithm for general array.
```
Comment (rgrinberg) [inline packages/levenshtein/levenshtein.1.1.3/descr]: I think it's recommended to have a new line between the short and long descriptions

File: packages/levenshtein/levenshtein.1.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version: "1.1.3"
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://bitbucket.org/camlspotter/ocaml_levenshtein/"
+bug-reports:
+  "https://bitbucket.org/camlspotter/ocaml_levenshtein/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ocaml_levenshtein"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocamlfind" {build}
```
Comment (rgrinberg) [inline packages/levenshtein/levenshtein.1.1.3/opam]: Same comment about ocamlfind. It's not necessary.

File: packages/levenshtein/levenshtein.1.1.3/opam [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "jbuilder" {build}
+  "ppx_test" {>= "1.6.0"}
+]
+available: [ocaml-version >= "4.02.1"]
```
Comment (rgrinberg) [inline packages/levenshtein/levenshtein.1.1.3/opam]: Does jbuilder even support `4.02.1`? Didn't some key ppx details change between `4.02.1` and `4.02.3`? Seems unlikely that you would support both between ppx_test.

Comment (mseri) [reply packages/levenshtein/levenshtein.1.1.3/opam]: @camlspotter have you tested a build on `4.02.1`?

Comment (rgrinberg) [reply packages/levenshtein/levenshtein.1.1.3/opam]: Probably not worth delaying the release for this. Nobody uses 4.02.1 and the constraint should really go on ppx_test anyway.

**Pull request merged**